### PR TITLE
user profile config added

### PR DESCRIPTION
**Pull request by RobertoCervera**
user profile config added

### CODE REVIEW

File: src/user/App.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Route, Switch } from "react-router-dom";
+import Profile from "./views//Profile";
+import { NavBar } from "src/partner/modules/NavBar";
+
+interface IUserAppProps {
+  match: {
```
Comment (mishelashala) [inline src/user/App.tsx]: create an interface `IReactRouterProps` and add it under `common`

File: src/user/views/Profile.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            <Input type="text" name="name" value={name} onChange={handleChangeName} />
+          </Fieldset>
+          <SaveButton>Save</SaveButton>
+        </ListItem>
+      </ProfileContainer>
+    </div>
+  );
+};
+
+export default Profile;
```
Comment (mishelashala) [inline src/user/views/Profile.tsx]: favor `named exports` over `export default`, [see](https://github.com/ksquarelabsmx/react-guidelines#modules)

and add an `index.ts` file that exports the `Profile` component.

**Review by mishelashala — CHANGES_REQUESTED** [review]

File: src/user/App.tsx (hunk 1 above)
Comment (RobertoCervera) [reply src/user/App.tsx]: is common going to be the shared directory?

Comment (mishelashala) [reply src/user/App.tsx]: yep, it is gonna be

File: src/partner/App.tsx [hunk 3]
```diff
@@ -3,14 +3,9 @@ import { Route, Switch } from "react-router-dom";
 import { CurrentEventsView, EventView, PastEventsView } from "./views";
 import { NavBar } from "./modules/NavBar";
 import { NotificationContext } from "../providers/";
+import { IReactRouterProps } from "src/commun/interfaces";
```
Comment (mishelashala) [inline src/partner/App.tsx]: `common`, not `commun`

**Review by mishelashala — CHANGES_REQUESTED** [review]

**Review by mishelashala — APPROVED** [review]

**Pull request merged**